Tests/test_image.py

```diff
@@ -1014,6 +1014,21 @@ def test_fli_overrun2(self):
             except OSError as e:
                 assert str(e) == "buffer overrun when reading image file"
 
+    @pytest.fixture(scope="function")
+    def inject_caplog(self, caplog):
+        self._caplog = caplog
+
+    @pytest.mark.usefixtures("inject_caplog")
+    def test_close_graceful(self):
```

Comment:
Why not just use the caplog fixture directly?

Reply:
I think I've had the same idea in https://github.com/RaphaelVRossi/Pillow/pull/1

Reply:
@radarhere already fixed this! Thanks again.